cobalt/h5vcc/h5vcc_settings.h

```diff
@@ -58,6 +58,11 @@ class H5vccSettings : public script::Wrappable {
   // invalid or not set to the expected value.
   bool Set(const std::string& name, SetValueType value) const;
 
+  void SetPersistentSettingAsInt(const std::string& key, int value) const;
```

Comment:
I'm unclear what this PR fixes. What's the intent here?

Reply:
The goal here is to enable javascript tests that set and read back persistent values to validate the settings and better understand the issues plaguing persistent settings.

Reply:
In what scenario can someone call this method and persistent_settings_ is not initialized? It's an arg to the constructor and constructed before H5vccSettings is. Do you think there's a race condition with destroying persistent settings and h5vcc settings?

Reply:
Hmm yeah you're right. persistent_settings_ is constructed in application.cc before browser_module which constructs h5vcc_setting so the only concern would be a race condition with the destructors.